docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -218,7 +220,17 @@ tcld login
 tcld apikey delete --id <api-key-id>
 ```
 
-## Manage API Keys for Service Accounts {#serviceaccount-api-keys}
+### Rotate an API key
+
+Temporal API keys automatically expire.
+Their maximum life is 90 days.
+Follow these steps to rotate API keys:
+
+1. Delete the existing key.
```

Comment:
They need to create a new key first and then allow both to be active at the same time. Then switch to the new key. Once they verify everything is switched to the new key and verified that the old key is no longer used they can delete it

Reply:
@mattkim Please confirm this is correct:

```
1. Create a new key.
   You may reuse key names if that helps.
1. Allow both keys to be active at the same time.
1. Switch to the new key.
1. Verify the new key works and the old key is no longer being used.
1. Delete the existing key.
```